src/main/java/de/tum/cit/aet/artemis/core/config/WebConfigurer.java

```diff
@@ -117,7 +117,8 @@ private String resolvePathPrefix() {
     public CorsFilter corsFilter() {
         UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
         CorsConfiguration config = jHipsterProperties.getCors();
-        if (config.getAllowedOrigins() != null && !config.getAllowedOrigins().isEmpty()) {
+        if ((config.getAllowedOrigins() != null && !config.getAllowedOrigins().isEmpty())
+            || (config.getAllowedOriginPatterns() != null && !config.getAllowedOriginPatterns().isEmpty())) {
```

Comment:
Consider using CollectionUtils.isEmpty() like JHipster's newer template does.
```suggestion
        if (!CollectionUtils.isEmpty(config.getAllowedOrigins()) || !CollectionUtils.isEmpty(config.getAllowedOriginPatterns())) {
```